torchaudio/io/_playback.py

```diff
@@ -0,0 +1,55 @@
+import torch
+import torchaudio
+from torchaudio.io import StreamWriter
+from typing import Optional
+
+
+def play_audio(
+    waveform: torch.Tensor,
+    sample_rate: Optional[float],
+    device: Optional[str] = "audiotoolbox",
+    channels_first: Optional[bool] = True,
```

Comment:
Let's simplify the function and remove this `channels_first` argument, and make **channel LAST** a requirement. it's easy for users to transpose the input.